src/fides/api/service/privacy_request/duplication_detection.py

```diff
@@ -38,9 +40,10 @@
 class DuplicateDetectionService:
     def __init__(self, db: Session):
         self.db = db
+        self.config = ConfigProxy(db).privacy_request_duplicate_detection
```

Comment:
I might makes this `_config` because people probably shouldn't be accessing it directly, and just hide whatever we are using in a public property (like `enabled`)